PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+I want to merge this change because...
+
+(Please mention all relevant issue numbers.)
+
+(If your changes affect the UI, providing "before" and "after" screenshots will
+greatly reduce the amount of work needed to review your work.)
+
+### Pull Request Checklist
+
+(Please keep this section. It will make maintainer's life easier.)
+
+1. [ ] Privileged views and APIs are guarded by proper permission checks.
+1. [ ] All visible strings are translated with proper context.
+1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
+1. [ ] Database queries are optimized and the number of queries is constant.
```

Comment:
It will be nice to catch that on CI (on existing code ofc). Can we do that?

Reply:
There's https://github.com/jmcarp/nplusone but I never tried to use it in an automated environment.